.github/workflows/nodejs.yml

```
@@ -0,0 +1,24 @@
+name: Node.js CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
```

Is there a way to just run on the latest version? Maintaining this list on each repository will be a chore and we aren't really concerned with old node versions.

---

Reading the [source](https://github.com/actions/setup-node/blob/d123f1054346d0acb63e9c1b205df110a951f36a/src/installer.ts#L128-L153), I think this will work since the versions are sorted before checking if they satisfy the version requested:

```suggestion
        node-version: latest
```